README.adoc

```diff
@@ -166,7 +166,7 @@ To directly bind your data to your custom types, you need to specify the
 variable name to be exactly the same as the key in the JSON document returned from the API.
 In case your variable name and key in JSON doc do not match, you can use `@JsonProperty`
 annotation to specify the exact key of the JSON document. (This example matches each
-variable name to a JSON key, so do not need that annotation here.)
+variable name to a JSON key, so we do not need that annotation here.)
```

Comment:
Guides all should be using direct, imperative speech ("you" no "we"). We (sic) might as well try and stick to that.

Reply:
So 'you' instead of 'we' and you are going to be happy, right?

Reply:
Yes, that's what I meant.